tests/model/test_suite_public_name.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from django.db.utils import IntegrityError
+
+SAME_SUITE_PUBLIC_NAME = 'same suite name'
+SAME_TEST_PUBLIC_NAME = 'same test name'
+SAME_SUITE = 'same suite'
+SAME_TEST = 'same test'
+
+
+@pytest.mark.parametrize("suite_public_name_1, suite_public_name_2,"
+                         "test_public_name_1, test_public_name_2,"
+                         "suite_1, suite_2, test_1, test_2", [
+                             [SAME_SUITE_PUBLIC_NAME, SAME_SUITE_PUBLIC_NAME,
+                              SAME_TEST_PUBLIC_NAME, SAME_TEST_PUBLIC_NAME,
+                              SAME_SUITE, SAME_SUITE, SAME_TEST, SAME_TEST],
+
+                             [SAME_SUITE_PUBLIC_NAME, SAME_SUITE_PUBLIC_NAME,
```

Comment:
Let's use `(` instead of `[`, so it's easier to read the parameters lists.

Reply:
OK